frontend/src/plugins/impl/SliderPlugin.tsx

```diff
@@ -71,49 +74,53 @@ const SliderComponent = ({
   }, [value]);
 
   return (
-    <Labeled
-      label={label}
-      id={id}
-      align={orientation === "horizontal" ? "left" : "top"}
-    >
-      <div
-        className={cn(
-          "flex items-center gap-2",
-          orientation === "vertical" &&
-            "items-end inline-flex justify-center self-center mx-2",
-        )}
+    <div className={fullWidth ? "my-3" : ""}>
```

Comment:
I'm not sure why this was needed -- but without it, when `fullWidth` is `True`, the bottom of the slider clips the output area ... @mscolnick